lib/operations/command_v2.js

```diff
@@ -2,13 +2,18 @@
 
 const OperationBase = require('./operation').OperationBase;
 const resolveReadPreference = require('../utils').resolveReadPreference;
+const ReadConcern = require('../read_concern');
+const WriteConcern = require('../write_concern');
 
 class CommandOperationV2 extends OperationBase {
   constructor(parent, options) {
     super(options);
 
     this.ns = parent.s.namespace.withCollection('$cmd');
-    this.readPreference = resolveReadPreference(parent, options);
+    this.readPreference = resolveReadPreference(parent, this.options);
+    this.readConcern = resolveReadConcern(parent, this.options);
+    this.writeConcern = resolveWriteConcern(parent, this.options);
+    this.explain = false;
```

Comment:
A little skeptical of `explain` being here. Why is this now a property of all commands?

Reply:
because a number of commands can be explained (aggregate, count, distinct, group, find, findAndModify, delete, and update). We could make a subclass called `ExplainableCommand` I suppose? I'm sort of leaving this kind of design work to follow-on, since the focus is getting retryable reads out the door.